Plugins/Object/Object.cpp

```diff
@@ -970,4 +971,17 @@ ArgumentStack Object::DoSpellLevelAbsorption(ArgumentStack&& args)
     return Services::Events::Arguments(retVal);
 }
 
+ArgumentStack Object::SetHasInventory(ArgumentStack&& args)
+{
+    if (auto *pPlaceable = Utils::AsNWSPlaceable(object(args)))
+    {
+        const auto hasInventory = Services::Events::ExtractArgument<int32_t>(args);
```

Comment:
Can just do something like `const auto hasInventory = !!Services::Events::ExtractArgument<int32_t>(args);` so you don't have to bother with the asserts